src/test/java/org/broadinstitute/hellbender/tools/PrintReadsCRAMIntegrationTest.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.read.SamAssertionUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class PrintReadsCRAMIntegrationTest extends CommandLineProgramTest{
```

Comment:
Could you move these tests into the existing `PrintReadsIntegrationTest` class (but keep them as separate test cases from the tests already there)?